bugbug/model.py

```diff
@@ -133,7 +138,6 @@ def train(self, importance_cutoff=0.15):
 
         # Evaluate results on the test set.
         y_pred = self.clf.predict(X_test)
-
         print(f"No confidence threshold - {len(y_test)} classified")
         print(metrics.confusion_matrix(y_test, y_pred, labels=class_names))
         print(classification_report_imbalanced(y_test, y_pred, labels=class_names))
```

Comment:
You can add metrics from this too (which will be the only metrics when CV is disabled).

Reply:
I've added the confusion matrix to the tracking metrics. Do you want to add also the metrics from  `classification_report_imbalanced`? If yes, I imagine it would be the raw metrics, not the formatted report, right?

Reply:
Yep!

Reply:
Filed #619 for this.